raycicmd/converter.go

```diff
@@ -8,6 +8,8 @@ import (
 )
 
 const windowsJobEnv = "WINDOWS"
+const macosJobEnv = "MACOS"
+const macosDenyFileRead = "/usr/local/etc/buildkite-agent/buildkite-agent.cfg"
```

Comment:
what I meant is that to define a:

````
var macosPlugin = []any{map[string]any{
	macosSandboxPlugin: map[string]string{
		"deny-file-read": macosDenyFileRead,
	},
}
````

but I guess this is also fine.

Reply:
w00t already merged